Demo/MozillaSocial-iOS/MoSoContent/LoginView.swift

```diff
@@ -0,0 +1,86 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+
+struct LoginView: View {
+    @ObservedObject var auth = AuthenticationService()
```

Comment:
one caveat with an `@ObservedObject` initialized in the view itself: this will be re-initialized every time the view is re-drawn, which could lead to unwanted behavior.
There are a couple of options to fix this:
1. Inject the object into the view
2. Use `@StateObject` instead of `@ObservableObject`

Reply:
I went for StateObject to keep the code encapsulated.